src/components/ProductAvatar/ProductAvatar.tsx

```diff
@@ -12,6 +12,8 @@ export interface ProductAvatarProps {
   logoUrl: string;
   /** The logo background color */
   logoBgColor?: string;
+  /** Alternate text for logo */
+  logoAltText?: string;
```

Comment:
```suggestion
  logoAltText: string;
```
If the alternate text is required for accessibility, it should be always required